### DISCUSSION TITLE
Default Signals per Workspace

### DISCUSSION THREAD

**Discussion by CrossBound** [datalust/seq-tickets#1228] [Ideas]
Currently it is possible under the user settings to select default signals, but I would like my default signals to be different depending on what workspace I'm under. Is that currently possible?

**Comment by nblumhardt** [comment]
Hi David,

Yes, this is implemented and shipped in 2020.5.5163:

https://github.com/datalust/seq-tickets/issues/770

If you've already upgraded and it's not showing up as expected, please let me know.

Best regards,
Nick
